app/controllers/cases_controller.rb

```diff
@@ -109,10 +109,19 @@ def filter
     redirect_to redirect_url
   end
 
+  def select_type
+    @case = Case.new
+  end
+
   def new
+    if FeatureSet.sars.enabled?
+      @type = params[:case][:type]
```

Comment:
We're still not using the type in the case creation, we're just making a standard Case

Reply:
Ya, there's a separate ticket to deal with case creation ... CT-1391, I think that using the case type can go there.